recipes/correct-ts-specifiers/src/is-dir.ts

```diff
@@ -0,0 +1,20 @@
+import { lstat } from 'node:fs/promises';
+
+import type { FSAbsolutePath, NodeError, ResolvedSpecifier, Specifier } from './index.d.ts';
+import { resolveSpecifier } from './resolve-specifier.ts';
+
+export async function isDir(parentPath: FSAbsolutePath | ResolvedSpecifier, specifier: Specifier) {
+	let resolvedSpecifier: ResolvedSpecifier;
+	try {
```

Comment:
```suggestion

	try {
```

Reply:
I keep these together because the `try` is merely for setting the `let` (which would otherwise be a `const`). For anything else, I do separate them.